src/tests/elementary/elm_test_check.c

```diff
@@ -196,10 +198,12 @@ EFL_START_TEST(elm_test_check_api_call)
 
    get_me_to_those_events(check1);
 
+#ifdef HAVE_ECTOR
    data.value = 0;
    click_object(data.check1);
    ecore_main_loop_iterate();
-   ck_assert_int_eq(data.value, 1);
+   ck_assert_int_eq(data.value, 1); // FIXME No more ifdef have_ector
```

Comment:
What's the problem here?

Reply:
Problem is events' callbacks aren't being called for Eo objects, so that `data.value` is never set.